### ISSUE TITLE
Do not publish src

### ISSUE THREAD

**Issue by TrejGun** [niuware/mui-rte#206]
## Checklist

- [ ] This is not covered in the [repository examples](https://github.com/niuware/mui-rte/tree/master/examples).
- [ ] I am using the package [latest version](https://github.com/niuware/mui-rte/releases/latest).
- [x] This is a bug report or an enhancement proposal.

## Description

I just realized that you are publishing `src` folder  together with `dist` folder, probably for types
Guys there is one huge downsite for this
My TS config is more strict than yours and for latest version i have error

```
../../node_modules/mui-rte/src/MUIRichTextEditor.tsx(114,6): error TS6196: 'TCustomRenderers' is declared but never used.
```

and therefore can't compile my project. Can you please build with tsc? this will give you code and types to publish as npm

in case you want to create umd/cjs you can still use rollup but there is no need to include this build in npm you can destribute it separatly using unpkg

**Comment by niuware** [comment]
This is fixed on version `1.26.1`.

**Comment by TrejGun** [comment]
thanks

**Comment by vilanz** [comment]
@niuware `src` is still being published in version `1.26.2`.

**Comment by TrejGun** [comment]
@vilanz you can use my fork
https://github.com/paktolus/mui-rte

**Comment by niuware** [comment]
> @niuware `src` is still being published in version `1.26.2`.

Gonna work on this as soon as I can 👍

**Comment by niuware** [comment]
The source files are removed from version [1.28.0](https://github.com/niuware/mui-rte/releases/tag/1.28.0).
